src/Microsoft.AspNetCore.Razor.OmniSharpPlugin/ProjectLoadListener.cs

```diff
@@ -78,12 +105,20 @@ public void ProjectLoaded(ProjectLoadedEventArgs args)
                     return;
                 }
 
-                var razorConfiguration = GetRazorConfiguration(args.ProjectInstance);
+                var razorConfiguration = GetRazorConfiguration(args.ProjectInstance, _projectConfigurationProviders);
+                var projectDirectory = args.ProjectInstance.GetPropertyValue(MSBuildProjectDirectoryPropertyName);
+                var fileSystem = RazorProjectFileSystem.Create(projectDirectory);
```

Comment:
Thought about adding tests for all these guys but it ends up just being a boatload of ceremony with not a lot of value.